openapi.yaml

```
@@ -129,6 +127,10 @@ components:
       $ref: './v2/components/schemas/PlugRef.yaml'
     Progress:
       $ref: './v2/components/schemas/Progress.yaml'
+    Prompt:
+      $ref: './v2/components/schemas/Prompt.yaml'
+    PromptReply:
+      $ref: './v2/components/schemas/Prompt.yaml'
```

```suggestion
      $ref: './v2/components/schemas/PromptReply.yaml'
```

Also, do you need to list the `CameraConstraints`/`PromptConstraintsCamera`, etc. here as well?